doc/optparse/tutorial.rdoc

```diff
@@ -55,7 +55,14 @@ The class also has method #help, which displays automatically-generated help tex
 - {Argument Converters}[#label-Argument+Converters]
 - {Help}[#label-Help]
 - {Top List and Base List}[#label-Top+List+and+Base+List]
-- {Methods for Defining Options}[#label-Methods+for+Defining+Options]
+- {Defining Options}[#label-Defining+Options]
```

Comment:
This label conflicts with [another line](https://github.com/ruby/optparse/pull/24/files#diff-847930dd3ac40523c1aedb290ee1acb7ef0d8b4611be6fef9cbdc70076d8c661R35).